maven-surefire-plugin/pom.xml

```diff
@@ -43,6 +43,13 @@
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
Is there a plexus dependency we can remove here now?

Reply:
I wasn't sure with an initial glance, will look more deeply now.